bfvmm/src/entry/entry.cpp

```diff
@@ -28,31 +28,35 @@
 //
 
 #include <bfgsl.h>
-#include <bfdebug.h>
+#include <bfarch.h>
 #include <bfvcpuid.h>
 #include <bfobject.h>
 #include <bfexports.h>
 #include <bfsupport.h>
-#include <bfexception.h>
-#include <bftypes.h>
 #include <bfcallonce.h>
+#include <bfexception.h>
 
 #include <vcpu/vcpu_manager.h>
 #include <debug/debug_ring/debug_ring.h>
 #include <memory_manager/memory_manager.h>
 
-#include "entry/entry.h"
-#include "bfvmm.h"
-
 static bfn::once_flag g_init_flag;
 
-extern "C" int64_t
-private_init(void)
-{ return ENTRY_SUCCESS; }
+void
+WEAK_SYM global_init()
```

Comment:
Is there documentation explaining these new extension functions?

Reply:
No... we need a place to put that. The entry code is not the right place. A future PR absolutely will have to document all of these functions. Any volunteers?

Reply:
I can do that. I've started some documentation on my cmake branch